crates/extensions/tedge_config_manager/src/config.rs

```diff
@@ -146,17 +150,29 @@ impl Borrow<str> for FileEntry {
 }
 
 impl FileEntry {
+    #[allow(clippy::too_many_arguments)]
```

Comment:
I would consider to pass `PermissionEntry` structs instead of creating those from argument lists (indeed too large and error prone).